Makefile

```diff
@@ -10,7 +10,7 @@ setup-python:
 	brew list python@3.9 || brew install python@3.9
 
 setup-poetry:
-	which poetry || curl -sSL https://install.python-poetry.org | python - -y --version 1.5.1
+	python -m scripts.install_poetry -y --version 1.5.1
```

Comment:
In a future dev exp task, I will see if we could just use the latest version of poetry instead of pinning to 1.5.1. Fern autogenerated 1.5.1 by default